config/m4/rdmacm.m4

```diff
@@ -8,9 +8,10 @@
 # Check for RDMACM support
 #
 rdmacm_happy="no"
+param_set="no"
 AC_ARG_WITH([rdmacm],
            [AS_HELP_STRING([--with-rdmacm=(DIR)], [Enable the use of RDMACM (default is yes).])],
-           [], [with_rdmacm=yes])
+           [param_set=yes], [with_rdmacm=yes])
```

Comment:
pls set [with_rdmacm=guess] in case of "action-if-not-given", and use it to decide if need to fail or not
see config/m4/cuda.m4 for an example

Reply:
@yosefe why not ```rdma_cm=auto``` to set automatic mode? it is more "native" for auto-tools :)